src/client/react-environment.js

```diff
@@ -0,0 +1,26 @@
+const getEnvJsContent = env => {
```

Comment:
REQ: Wie besprochen sollen die Settings aus dem .NET API bezogen werden, da wir sonst die Settings doppelt führen. Da der Client immer mit dem API deployed wird und ohne auch nicht ohne funktioniert, macht die strikte Trennung bzw. die Doppelspurigkeit nicht viel Sinn.

Reply:
Ich werde dies in einem separaten PR umsetzten, da sich dabei auch die Dev/Cypress Umgebung etwas ändern muss:
https://github.com/geoadmin/suite-bdms/issues/938

Reply:
Hat `network_mode: host` nicht funktioniert?